### PR TITLE
Crowd Strike Falcon X - bug with uploading file

### PR DESCRIPTION
**Pull request by ohaim1008**
## Status
- [x] ready - customer verified

Changing how we upload file - was verified with customer.
Adding a section in the read me about how to upload a file - customer wasn't sure and we get bugs due to miss understanding there.

## Related Issues
fixes: https://github.com/demisto/etc/issues/34179
fixes: https://github.com/demisto/etc/issues/34597

## Minimum version of Demisto
- [x] 5.0.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CrowdStrikeFalconX/ReleaseNotes/1_1_1.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### CrowdStrike Falcon X
+- Fix an issue with where *cs-fx-upload-file* didn't uploaded exe and dll files correctly.
```
Comment (yuvalbenshalom) [inline Packs/CrowdStrikeFalconX/ReleaseNotes/1_1_1.md:4]:
```suggestion
- Fix an issue where **cs-fx-upload-file** failed to upload .exe and .dll files correctly.
```

File: Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py [hunk 2]
```diff
@@ -215,19 +214,14 @@ def upload_file(
...
-        finally:
-            shutil.rmtree(file_name, ignore_errors=True)
+        get_file_path_res = demisto.getFilePath(file)
+        file_path = get_file_path_res["path"]
+        file_name = get_file_path_res["name"]
+
+        url_suffix = f"/samples/entities/samples/v2?file_name={file_name}&is_confidential={is_confidential}" \
+                     f"&comment={comment}"
+        self._headers['Content-Type'] = 'application/octet-stream'
+        return self._http_request("POST", url_suffix, data=open(file_path, 'rb'))
```
Comment (yuvalbenshalom) [inline Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py:224]: where do you close the file?

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
see my comments

**Review by yuvalbenshalom — APPROVED** [review]

File: Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/README.md [hunk 3]
```diff
@@ -13,6 +13,16 @@ Use the CrowdStrike Falcon X integration to submit files, file hashes, URLs, and
 | proxy | Use system proxy settings | False |
 
 4. Click **Test** to validate the URLs, token, and connection.
+
+
+##Uploading a file to the sandbox
+Please pay attention that there are 2 ways to upload a file to the sandbox.
```
Comment (amshamah419) [inline Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/README.md:19]:
```suggestion
There are 2 ways to upload a file to the sandbox.
```

File: Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/README.md [hunk 4]
```diff
@@ -13,6 +13,16 @@ Use the CrowdStrike Falcon X integration to submit files, file hashes, URLs, and
...
+1. Using ***cs-fx-upload-file*** command with **submit_file=yes**.
```
Comment (amshamah419) [inline Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/README.md:20]:
```suggestion
1. Using the ***cs-fx-upload-file*** command with **submit_file=yes**.
```

File: Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/README.md [hunk 5]
```diff
@@ -13,6 +13,16 @@ Use the CrowdStrike Falcon X integration to submit files, file hashes, URLs, and
...
+2. Using ***cs-fx-upload-file*** command and afterwards the ***cs-fx-submit-uploaded-file command***,
```
Comment (amshamah419) [inline Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/README.md:21]:
```suggestion
2. Using the ***cs-fx-upload-file*** command and afterwards the ***cs-fx-submit-uploaded-file command***, 
```

File: Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/README.md [hunk 6]
```diff
@@ -13,6 +13,16 @@ Use the CrowdStrike Falcon X integration to submit files, file hashes, URLs, and
...
 4. Click **Test** to validate the URLs, token, and connection.
+
+
+##Uploading a file to the sandbox
+Please pay attention that there are 2 ways to upload a file to the sandbox.
+1. Using ***cs-fx-upload-file*** command with **submit_file=yes**.
+2. Using ***cs-fx-upload-file*** command and afterwards the ***cs-fx-submit-uploaded-file command***, 
+in this option the sha256 identifier from the ***cs-fx-upload-file*** command output is the input to the ***cs-fx-submit-uploaded-file command***.
+
+For more information look at the commands documentation.
```
Comment (amshamah419) [inline Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/README.md:24]:
```suggestion
For more information review the documentation for the commands.
```

File: Packs/CrowdStrikeFalconX/ReleaseNotes/1_1_1.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### CrowdStrike Falcon X
+- Fix an issue where **cs-fx-upload-file** failed to upload .exe and .dll files correctly.
```
Comment (amshamah419) [inline Packs/CrowdStrikeFalconX/ReleaseNotes/1_1_1.md:4]:
```suggestion
- Fix an issue where ***cs-fx-upload-file*** failed to upload `.exe` and `.dll` files correctly.
```

**Pull request merged**